examples/functions/01_basic_oracle/src/actions/set_enclaves.rs

```diff
@@ -38,7 +38,7 @@ impl SetEnclaves<'_> {
 
         if let Some(enclaves) = params.mr_enclaves.clone() {
             if !enclaves.is_empty() {
-                program.mr_enclaves = parse_mr_enclaves(&enclaves)?;
+                // program.mr_enclaves = parse_mr_enclaves(&enclaves)?;
```

Comment:
?

Reply:
previously i was storing an array of mr_enclaves on program_state to validate the quote. but we made some changes to the attestation_program so we can now store an array of enclaves on the function account directly